### DISCUSSION TITLE
Can't SSH to update octoprint.

### DISCUSSION THREAD

**Discussion by tylamp** [Guilouz/Creality-Helper-Script-Wiki#663] [Issues]
Hello! 
I was using my printers yesterday and now I cannot connect to either my K1 or K1 Max because I'm getting this error in Orca:

"Could not connect to OctoPrint: curl:Couldn't connect to server:
Failed to connect to xxx.xxx.xx.xxx port 4408: Bad access
[Error 7]
Note: OctoPrint version 1.1.0 or higher is required."

I cannot SSH using putty and I cannot access the web interface using the browser with or without the :4408, I keep just getting access denied or unable to connect.

Both printers are rooted using the helper script and neither have had connection issues until this point. Is there anything that I can do, or do I have to factory reset my devices and re-root?

**Comment by lukas-frey** [comment]
I have the same problem, did you fix the issue? The wiki doesn't say anything about octoprint - should it have been installed automatically? Do I need to install it?

This used to work perfectly a year or so back but now I updated the firmware and reinstalled the helper script (which now has a completely different wiki etc.) and now it doesn't work.

**Reply by tylamp** [reply]
This still happens to me occasionally when the computer goes to sleep while the printer is still on (at least I think that’s the issue), and always while using or after using a vpn. Sometimes I can restart the printer firmware using guppyscreen UI and it reconnects, usually I have to restart my computer and it will work.

**Reply by lukas-frey** [reply]
Would it be possible for you to screenshot this part in your mainsail / fluidd interface? 
<img width="566" alt="image" src="https://github.com/user-attachments/assets/6a92c946-5ec8-430d-afdc-8aa997d94fed" />

I'm struggling to find a solution to this.

Totally out of NOWHERE orcaslicer successfully connected to my printer yesterday. Today I tried again and I'm back at this weird warning. 

So I have no idea if there's something missing installed, if a config is invalid or what. Because it's just weird that in 1 out of a 100 tries it suddenly works and then it stops working again.

**Reply by tylamp** [reply]
Yeah I genuinely have no clue what is happening or why. The only thing that usually works is restarting the computer and the printer. My issues have been becoming less frequent tho, sorry you're still struggling.
